src/starterConfig.js

```diff
@@ -0,0 +1,43 @@
+import PROJ from './utils/EPSG3857';
+import CONF from './config';
+
+export default {
+  layers: [
+    {
+      name: 'Basemap',
+      visible: true,
+      isBaseLayer: true,
+      data: {
+        type: 'wmts',
+        url:
+          `${CONF.tilesUrl}/wmts/netzkarte_relief_grey_webmercator/` +
+          'webmercator/{TileMatrix}/{TileCol}/{TileRow}.png',
+        matrixSet: 'webmercator',
+        requestEncoding: 'REST',
+        projectionExtent: PROJ.extent,
+        resolutions: [
+          156543.033928,
```

Comment:
You could use the wmts Resolutions from src/utils/WMTSTileGrid3857.js

Reply:
If I remember that correctly, @mhaertwig suggested to integrate the WMTSTileGrid configuration into the new config where also the layers are defined. Is that correct? What was the reason?
I have just integrated these two WMTSTileGrid files into the new layer config file. So, the information does not get lost.

Reply:
I don't know what the reason were.